spec/integration/build-swift/after/Example/Example.xcodeproj.yaml

```diff
@@ -0,0 +1,168 @@
+
+File References:
```

Comment:
ah I see. Oh well one could always hope :) Are you primarily using it to check for changes in xcodeproj?